model_bump_test.go

```diff
@@ -70,260 +99,512 @@ func TestBUMPModel_CalculateBUMP(t *testing.T) {
 		}
 
 		// when
-		bump, err := CalculateMergedBUMP(merkleProofs)
+		bump, err := CalculateMergedBUMP(bumps)
 
 		// then
 		assert.NoError(t, err)
 		assert.Equal(t, expectedBUMP, bump)
 	})
 
-	t.Run("Slice of Merkle Proofs", func(t *testing.T) {
+	// Cannot test unless we get real BUMPs - Merkle Proofs don't match in these examples
+	// For test on real data look at the last test in this file.
```

Comment:
If we have such tests already shouldn't we just remove this commented one?

Reply:
You're right, done